src/WebSdk/Publish/Tasks/WebConfigTransform.cs

```diff
@@ -98,12 +98,6 @@ private static void TransformAspNetCore(XElement aspNetCoreElement, string appNa
                 appPath = Path.ChangeExtension(appPath, !string.IsNullOrWhiteSpace(extension) ? extension : null);
                 aspNetCoreElement.SetAttributeValue("processPath", appPath);
             }
-            // For Apps targeting .NET Framework, the extension is always exe. RID is not set for .NETFramework apps with PlatformType set to AnyCPU.
```

Comment:
This was not always true, non-SDK style .NET Framework MVC applications compile with *.dll's, *.pdb's, *.config's ext files by default only. Those projects when migrated in place (Target Framework wise) to SDK style would refuse to build when set to Exe because they lack a Main and therefore would still need to be set to dll in order to build and run correctly.

Due to that and the fact that my changed code would prevent this from ever being called under .NET Framework, this made this check unneeded hence the cleanup of it.